lua/dapui/init.lua

```diff
@@ -114,10 +114,22 @@ function dapui.setup(user_config)
   ui_state = UIState()
   ui_state:attach(dap)
 
+  expand_lines = config.sidebar().expand_lines
```

Comment:
This setting is not specific to the sidebar, it is being applied to all of the elements. Move it to the top level of the config